cloudformationTemplates/asyncCustomerChatUX/README.md

```diff
@@ -126,6 +126,24 @@ If you want to enable sending attachments for Amazon Connect Chat the customer c
     },successHandler, failureHandler)
 ```
 
+## Enabling message receipts
+
+To render/send message receipts, enable the feature [in the admin console](https://docs.aws.amazon.com/connect/latest/adminguide/message-receipts.html), and update [`chat-interface`](https://github.com/amazon-connect/amazon-connect-chat-interface#message-receipts) to generate latest webpack bundle.
+
```

Comment:
`Ensure that the ChatJs feature configuration is set as well:` - best to add this to the section for `disabling message-receipts`

Also, you can explain about throttling in a separate section.